### DISCUSSION TITLE
Features from the "old" Battlesnake Design?

### DISCUSSION THREAD

**Discussion by jpierer** [BattlesnakeOfficial/feedback#240] [Q&A] [answered]
Hello from Germany.
A short question about the current, for me new Battlesnake design:

Is there no link to Github or a website on the profile pages anymore?
I always found it very exciting to see who is behind the snake/s and what other repositories, websites etc. the person have.
So i liked the Link (Github Repo/Website etc.) feature a lot.

I also liked the graph view (latency, length) of each snake in the games view. Unfortunately, this is also no longer available.

Are there already plans if these two features will be included in the "new" Battlesnake design?

**Comment by bvanvugt** [comment, marked as answer]
Hey! These are good questions, thanks for posting :)

Re: GitHub profile links - hmm that's an oversight on our part. We should definitely be able to add those back in.

Re: Graphs - Interesting! You're the first to mention the length/latency graphs, which I always thought was cool, but we were never sure if they were actually useful. That one is a lot less likely to come back soon, just because we're so limited on dev time these days and it's a complicated thing to power. 

That said, the data is still available, and there may be others in the community that want to help get those live again.
